packages/@okta/vuepress-site/docs/guides/sign-users-out/define-signout-callback/react-native/addbaseuri.md

```diff
@@ -0,0 +1,17 @@
+Signing out of Okta requires the app to open a browser and navigate to the [end session endpoint](https://developer.okta.com/docs/reference/api/oidc/#logout). Okta destroys the user's session and immediately redirects back to your application. To do this, you must define a callback route for the sign-out process.
+
+Make sure that the `endSessionRedirectUri` configured at `createConfig` is the same as the one defined in the **Logout redirect URI** section. For example if you are initializing your client via a config file:
```

Comment:
Might be worth adding a little more context here: "**Logout redirect URI** section of the okta admin dashboard" or similar.